test/res/xml/config.xml

```diff
@@ -35,6 +35,7 @@
     <preference name="useBrowserHistory" value="true" />
     <preference name="exit-on-suspend" value="false" />
     <preference name="showTitle" value="true" />
+    <preference name="BackgroundColor" value="true" />
```

Comment:
If you want to fix "empty string" crash shouldn't you do the test with an empty string?